### ISSUE TITLE
[Research] Dot Indicator: Draft research plan/usability study

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#66479]
## Background
It is likely that we will need to do some kind of usability testing on the dot indicator (and related/potential options). This ticket is to work on an initial/draft research plan for a usability study. At a high level it should outline our most desired path incorporating feedback by testing on My VA and MHV landing page, but with the knowledge that it could end up being a study limited to just My VA.

## Tasks
- [x] Create draft research plan 
- [x] Share/discuss with PO and PM

## Acceptance Criteria
- [ ] Initial research plan for usability testing is drafted

**Comment by aagosto90** [comment]
Hi @Samara-Strauss and @ajakabcin ! Here's a draft of the dot/badge indicator research plan for your review. No rush on this - Just trying to get the research goals and questions solidified by the end of the sprint (but I did the whole plan excluding the dates at the end). Should we have someone from the MHV team review this as well?

A few things:
- I referenced Cara's [Mural board](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1695832058564/28ef4d0bdc3c62e6fe45714639efceefe554c8b5?sender=ud5c82d652d231345efaf6060) when writing this research plan, but a lot of those questions are around what types of notifications and how many Veterans want to receive, which seems out of scope for this particular study.
- I think we can do an observer sign-up sheet in Slack for this one so people from the MHV team can observe if they’d like to, and maybe others.
- The most important recruitment criteria to me is the AT and colorblind users so I kept the other criteria light. I did some so we don't end up with all men or all white people. 
- I thought with the accessibility criteria being so specific, we could forego asking for Veterans who use different types of benefits and therefore would find different types of notifications useful. That'd be nice to have though, so maybe I can list it as not a "must."

**Comment by Samara-Strauss** [comment]
> The most important recruitment criteria to me is the AT and colorblind users so I kept the other criteria light. I did some so we don't end up with all men or all white people.

Sounds good to me!

> Total Requested: 12 Completed Sessions Needed: 8 User type: Veterans only

I can't tell what you are saying here. Do you need 12 or 8 people? EIther way, I would think you probably will start seeeing the same things after ~6 people. Thoughts?

> Session length: 45 minutes; For the screen reader session, please schedule it for 75 minutes

It doesn't seem like we have that much to discuss/test, but let's see when you have the conversation guide together. I would assume we could schedule sessions for much less time than this, but we can always adjust before submitting to Perigean.

**Comment by aagosto90** [comment]
Thank you for the feedback @Samara-Strauss ! 

I intended to shoot for 8 completed sessions, so therefore I'd be asking Perigean to recruit 12 people to account for no-shows. I'm thinking 8 because I want to speak with multiple colorblind and screen reader users (I'd say at least 2 of each) and when I meet with screen reader users, I'll be focusing on a different set of questions than with sighted users. I'd like to be able to ask the other 6 sighted people similar questions (with some additional for colorblind users) so I can track any trends among them. Does that all sound okay to you?

I changed the sessions to 30 minutes each, 60 minutes for screen reader users, but we'll finalize that once the convo guide is ready as you noted.
